modules/programs/k9s.nix

```
@@ -8,7 +8,7 @@ let
   yamlFormat = pkgs.formats.yaml { };
 
 in {
-  meta.maintainers = [ hm.maintainers.katexochen ];
+  meta.maintainers = [ hm.maintainers.katexochen maintainers.liyangau ];
```

```suggestion
  meta.maintainers = with maintainers;  [ katexochen liyangau ];
```
Should work nowadays, too. ;)